packages/harmony/src/components/input/TextInput/TextInput.module.css

```diff
@@ -0,0 +1,149 @@
+/* The absolute root of the entire component, including assistive text */
+.root {
+  /* This isolates z-index to only affect this component */
+  isolation: isolate;
+}
+
+/* The inputRoot is the root container of all things that aren't assistive text */
+.inputRoot {
+  display: flex;
+  align-items: center;
+  /* Dont need Y padding since the flex centering takes care of it */
+  padding: 0 var(--harmony-spacing-l);
+  /* The gap spaces the content around the icons */
+  border: 1px solid var(--harmony-border-default);
+  border-radius: var(--harmony-border-radius-s);
+  background-color: var(--harmony-bg-surface-1);
+  transition: border ease-in-out 0.1s;
+  box-sizing: border-box;
+  & svg path {
+    fill: var(--harmony-text-subdued);
+  }
+}
+
+/* hover border color */
+.inputRoot:hover {
+  border-color: var(--harmony-border-strong);
+}
+
+/* small input size */
+.small {
+  /* gap between icons & content */
+  gap: var(--harmony-spacing-s);
+  font-size: var(--harmony-font-s);
+  height: 34px;
+}
+
+/* default input size */
+.default {
+  /* gap between icons & content */
+  gap: var(--harmony-spacing-m);
+  font-size: var(--harmony-font-m);
+  height: 64px;
+}
+
+/* focused border color */
+.focused,
+.focused:hover {
+  border-color: var(--harmony-secondary);
+}
+
+/* warning border color */
+.warning,
+.warning.focused,
+.warning:hover {
+  border-color: var(--harmony-orange);
+}
+
+/* error border color */
+.error,
+.error.focused,
+.error:hover {
+  border-color: var(--harmony-red);
+}
+
+/* disabled styles */
+.disabled {
+  cursor: not-allowed;
+  & * {
+    cursor: not-allowed;
+  }
+}
+
+/* Actual input element */
+.inputElement {
+  padding: 0;
+  overflow: visible;
+  width: 100%;
+  height: 100%;
+  outline: 0;
+  border: 0;
+  background: none;
+  font-weight: var(--harmony-font-medium);
+  color: var(--harmony-text-default);
+  text-overflow: ellipsis;
```

Comment:
Yeah I actually forgot that I threw this in there 😅 , meant to ask for feedback on it. I also need to see if design has a preference. Since by default `input` doesn't do any form of multiline wrapping the default behavior just cuts off.
Personally I would advocate for ellipsis cause it gives a better visual indicator that there is more text than the "cut off".
Here's with ellipsis
![2023-10-18 18 19 58](https://github.com/AudiusProject/audius-protocol/assets/6711655/f9961d13-4632-40e7-ad26-b392a29f34f4)
Without ellipsis
![2023-10-18 18 21 24](https://github.com/AudiusProject/audius-protocol/assets/6711655/0402e34d-8c16-41f2-bf1b-ce4f4748fd4f)

Reply:
Side note, this made me check the label too and I think we could get away with growing the height with the container using a min-height instead of height, so now I did this
![image](https://github.com/AudiusProject/audius-protocol/assets/6711655/99dd63e3-83c2-4ee9-9642-78e42cd78820)

Reply:
ohh the ellipsis is only visible when unfocused. That's much more reasonable. Nice

Idk how I feel about the multiline label. I'd defer to design

Reply:
Design said ellipsis overflow on both, so we're good there 👍